php-frontend/src/main/java/org/sonar/plugins/php/api/tree/declaration/FunctionDeclarationTree.java

```diff
@@ -37,7 +44,7 @@
   IdentifierTree name();
 
   @Override
-  ParameterListTree parameters();
+  ArgumentListTree parameters();
 
   BlockTree body();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1161)